packages/widget-chat/src/components/icon/styles.css

```diff
@@ -0,0 +1,6 @@
+.icon {
+  font-family: 'Atlantic Neo Icons';
+  font-size: 18px;
```

Comment:
I think this is fine for now... we probably will need to change this into a prop at some point. There are also some cases where the icons have transforms (e.g. rotate 90 degrees) instead of just having another character represent the other version. That will also need to be handled in the icon component.